src/components/SettlementButton.js

```diff
@@ -131,7 +132,7 @@ class SettlementButton extends React.Component {
             >
                 {triggerKYCFlow => (
                     <ButtonWithMenu
-                        isDisabled={this.props.isDisabled}
+                        isDisabled={this.props.isDisabled || this.props.network.isOffline}
```

Comment:
Ah this seems redundant in this usage here

https://github.com/Expensify/App/blob/11a1f0debac2dfee461e6b71751c3b3bcfb9f2cf/src/components/IOUConfirmationList.js#L387-L390

Because `shouldDisableButton` is already looking at `this.props.network.isOffline`

https://github.com/Expensify/App/blob/11a1f0debac2dfee461e6b71751c3b3bcfb9f2cf/src/components/IOUConfirmationList.js#L351-L352